Popstation/Popstation.Repack.cs

```diff
@@ -59,7 +59,7 @@ public bool Repack(ConvertOptions options, CancellationToken cancellationToken)
                     Directory.Delete(path, true);
                 }
 
-                return cancellationToken.IsCancellationRequested;
+                return cancellationToken.IsCancellationRequested == false;
```

Comment:
How about use this instead?

return !cancellationToken.IsCancellationRequested

Reply:
When I was writing this code I thought it would be easier to read if I used `==`, but then I looked again and saw that `! ` is more readable, so I adopted it.

I also made some minor indentation corrections.

Thank you!